llarp/link/link_manager.hpp

```diff
@@ -345,13 +345,12 @@ namespace llarp
 
     // these requests are direct, i.e. not over a path;
     // the rest are relay->relay
-    // TODO: new RC fetch endpoint (which will be both client->relay and relay->relay)
     std::unordered_map<
         std::string_view,
-        void (LinkManager::*)(std::string_view body, std::function<void(std::string)> respond)>
+        void (LinkManager::*)(oxen::quic::message m)>
         direct_requests = {
-            {"publish_intro"sv, &LinkManager::handle_publish_intro},
-            {"find_intro"sv, &LinkManager::handle_find_intro}};
+            {"fetch_rcs"sv, &LinkManager::handle_fetch_rcs},
+            {"fetch_router_ids"sv, &LinkManager::handle_fetch_router_ids}};
```

Comment:
I'm all for smaller messages, eventually.  For the time being until we've done a good amount of testing, I would prefer the more descriptive keys if only because we'll be poring over log files and dumping bt-encoded containers to figure out where we went wrong and having it be human readable takes precedence for that purpose, imo.  Definitely not gonna die on that hill though.

Reply:
agreed. but lets make sure to sit down and knock out that conversation sometime, i agree that after testing is a great time for it